sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2815,7 +2815,8 @@ object GpuOverrides {
     exec[DataWritingCommandExec](
       "Writing data",
       ExecChecks((TypeSig.commonCudfTypes +
-        TypeSig.DECIMAL.withPsNote(TypeEnum.DECIMAL, "Only supported for Parquet")).nested(),
+        TypeSig.DECIMAL.withPsNote(TypeEnum.DECIMAL, "Only supported for Parquet") +
+        TypeSig.STRUCT.withPsNote(TypeEnum.STRUCT, "Only supported for Parquet")).nested(),
```

Comment:
Are we missing a test?  When the struct support went in, the fact that we forgot to update this should have been caught by the test.

Reply:
Thanks for pointing this out @jlowe I remembered that we added struct support for `GpuInMemoryTableScanExec` and confused it with `DataWritingCommandExec`. We have to turn this on as part of this PR. I have updated the PR name to reflect it